README.md

```diff
@@ -86,3 +86,55 @@ npx simple-git-hooks
 ```
 
 If linting fails, the commit will be blocked until issues are resolved.
+
+
+## Data Processing
+
+### Overview
+
+Offical Data Sources ───────────────────► Source Data ──────────────► Extracted Data ───────────────► Transformed Data ────────────► Formatted Data
+     (URLs)               Fetch               │        [Optional]           ▲            Transform                        Format
+                                              │          Extract            │
+                                              │                             │
+                                              └─────────────────────────────┘
+#### Official Data Sources
```

Comment:
We could add a pre-processing step to get the list of urls. (something like firecrawl maybe)

Reply:
Cool! haven't seen firecrawl before. I like it. My goal here was to get us to a point where we have the sources in version control. To be able to ensure transparency, as well as making refreshes for future years easier.

From what I understand this would be a step to take 'high level source', and potential find, scrape, fetch, data without having to point directly to the assets. If so, perhaps something to consider in a future iteration.